tests/test_crypto_spread_menu.py

```
@@ -0,0 +1,54 @@
+    
+    read_data = json.loads(fake_pid_file.read_text(encoding="utf-8"))
+    assert read_data["strategy"] == "crypto-spread"
+    assert read_data["dash"]["pid"] == 12345
+    assert read_data["dash"]["started_ticks"] == 638600000000000000
+    assert read_data["dash"]["port"] == 8802
+
+
+def test_menu_script_status_execution():
+    """Test executing scripts/crypto-spread-menu.ps1 status via pwsh -NoProfile."""
+    cmd = ["pwsh", "-NoProfile", "-Command", f"$env:OutputEncoding=[System.Text.Encoding]::UTF8; & '{SCRIPT_PATH}' status"]
+    res = subprocess.run(cmd, cwd=str(ROOT), capture_output=True, encoding="utf-8", errors="replace", timeout=15)
```

Ruff (0.16.3)

[error] `subprocess` call: check for execution of untrusted input

(S603)